kroxylicious-operator/src/test/resources/DerivedResourcesTest/clusterref-resource-not-found/out-Deployment-example.yaml

```diff
@@ -65,6 +65,13 @@ spec:
             - mountPath: "/opt/kroxylicious/config/proxy-config.yaml"
               name: "config-volume"
               subPath: "proxy-config.yaml"
+          resources:
+            limits:
+              "cpu": "500m"
+              "memory": "512Mi"
+            requests:
+              "cpu": "500m"
+              "memory": "512Mi"
```

Comment:
Do we have a test in scope somewhere to make sure user specified limits are passed through?